eagleproject/core/templates/dashboard.html

```diff
@@ -605,6 +605,16 @@ <h2>Recent Activity</h2>
                             {% elif log.address == "0x3c5fe0a3922777343cbd67d3732fcdc9f2fa6f2f" %}
                                 CRV
                             {% endif %}
+                        {% elif log.topic_0 == "0x7d84a6263ae0d98d3329bd7b46bb4e8d6f98cd35a7adb45c274c8b7fd5ebd5e0" and log.address == "0x3cdd07c16614059e66344a7b579dab4f9516c0b6" %}
```

Comment:
Perhaps add the sign by hashing it (to sigs.py) and then use the named variable here for more readability? 
 `keccak(b"ProposalCreated(uint256,address,address[],uint256[],string[],bytes[],uint256,uint256,string)")`

Reply:
I think we can refactor this at some point later - because all of the dashboard topic checks use the hash directly. What do you think ?

Reply:
Fair enough